src/System.IO/tests/Stream/Stream.NopStream.cs

```diff
@@ -0,0 +1,48 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using Xunit;
+
+namespace System.IO.Tests
+{
+    public sealed class NopStream : Stream
```

Comment:
Why is this necessary?  Why can't you just use an existing stream like MemoryStream?